packages/vue-server-renderer/build.dev.js

```diff
@@ -8848,7 +8848,7 @@ function mapIdToFile (id, clientManifest) {
     fileIndices.forEach(function (index) {
       var file = clientManifest.all[index];
       // only include async files or non-js, non-css assets
-      if (clientManifest.async.indexOf(file) > -1 || !(/\.(js|css)($|\?)/.test(file))) {
+      if (clientManifest.async.indexOf(file) > -1 || (file && !(/\.(js|css)($|\?)/.test(file)))) {
```

Comment:
We can just drop if there is no `file`

```suggestion
      if (file && (clientManifest.async.indexOf(file) > -1 || !(/\.(js|css)($|\?)/.test(file))) {
```